content/css/docs.css

```diff
@@ -347,6 +347,12 @@ footer {
   line-height: 1.5;
 }
 
+.doc-content .alert {
```

Comment:
Could you add a comment line so we remember later why this is here and don't remove it:

```
// This is to prevent NOTE/admonition alert boxes from overlapping the table of contents at the top of a docs page.
```

Reply:
done